content/api/commands/intercept.md

```diff
@@ -2,73 +2,72 @@
 title: intercept
 ---
 
-Cypress routes all HTTP requests - including `XMLHttpRequest` (XHR) and `fetch` - through its proxy. Use `cy.intercept` to manage the behavior of these requests, including:
+`cy.intercept` can be used to capture or even change the behavior or contents of network traffic.
```

Comment:
For consistency with other API docs, instead of mentioning the API method like so:
> `cy.intercept` can be used to capture and even change the behavior or contents of network requests and responses.

Can you do something like this:
> Capture or change the behavior or contents of network requests and responses.